Mathlib/NumberTheory/Bernoulli.lean

```diff
@@ -88,7 +88,7 @@ theorem bernoulli'_spec (n : ℕ) :
 #align bernoulli'_spec bernoulli'_spec
 
 theorem bernoulli'_spec' (n : ℕ) :
-    (∑ k in antidiagonal n, ((k.1 + k.2).choose k.2 : ℚ) / (k.2 + 1) * bernoulli' k.1) = 1 := by
+    (∑ k in Nat.antidiagonal n, ((k.1 + k.2).choose k.2 : ℚ) / (k.2 + 1) * bernoulli' k.1) = 1 := by
```

Comment:
Note that after you make the typeclass change above, you'll be able to revert all of these, as there will be no `Nat.antidiagonal`.